packages/aws-cdk/lib/api/hotswap/lambda-functions.ts

```diff
@@ -1,3 +1,5 @@
+import { Writable } from 'stream';
+import * as archiver from 'archiver';
 import { flatMap } from '../../util';
```

Comment:
Can you also update the ReadMe file with this new capability? This section: https://github.com/aws/aws-cdk/blob/master/packages/aws-cdk/README.md#hotswap-deployments-for-faster-development